eng/SourceBuild.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <GitHubRepositoryName>llvm-project</GitHubRepositoryName>
     <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
+    <PackageLicenseFile>$(RepoRoot)/llvm/LICENSE.TXT</PackageLicenseFile>
```

Comment:
we're already setting this here: https://github.com/dotnet/llvm-project/blob/7de031a308e84d438576d47d2e24b47c00f3f22f/nuget/packaging.props#L6

I wonder why that isn't working in the SourceBuild case?

I'm fine with checking this in if it unblocks you though.

Reply:
Thanks, I'll write up an issue to look into this further but that would be good.

Reply:
Ah I didn't see that this was targetting the `objwriter/12.x` branch which is owned by the JIT team. We've recently merged the objwriter changes into main so for .NET8 there should only be one branch: https://github.com/dotnet/llvm-project/pull/287

Do you need it in earlier branches or can this be retargeted to main?